index.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Dyary's Portfolio</title>
+        <link rel="stylesheet" href="styles.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div class="container">
```

Comment:
could you add a <section/> tag instead of div here. it would be more semantically appropriate.